src/main/java/com/google/javascript/gents/CommentLinkingPass.java

```diff
@@ -65,7 +66,8 @@ public NodeComments getComments() {
   public void process(Node externs, Node root) {
     for (Node script : root.children()) {
       if (script.isScript()) {
-        List<Comment> comments = compiler.getComments(script.getSourceFileName());
+        ImmutableList<Comment> comments =
```

Comment:
I just followed the code path for my own curiosity. It turns out this will not need to superfluously copy the list of comments, because the underlying type is actually already an `ImmutableList` and `copyOf` returns (through many indirections) the `ImmutableList` itself in that case.

You could add a comment, in case somebody with the same curiosity finds this :-)